bookkeeper-server/src/main/java/org/apache/bookkeeper/proto/BookieRequestProcessor.java

```diff
@@ -264,6 +264,7 @@ int maxReadsInProgressCount() {
 
     @Override
     public void close() {
+        LOG.info("Closing RequestProcessor");
```

Comment:
Any additional info with this log to help observability or traceability? 
I am not sure logging this will help troubleshooting anything.

Reply:
That's a good idea @hsaputra I think we should holistically look at the improving observability across the entire shutdown sequence. Thoughts? @hsaputra @Vanlightly

Reply:
Actually, when I tested this myself, I added a log statement so I could see when it started and completed shutdown of the request processor. Given that it could block for a little bit.